libtfhe/core/arithmetic/big_torus.h

```diff
@@ -54,6 +54,8 @@ void sub(BigTorus *res, const BigTorus *a, const BigTorus *b, const ZModuleParam
  */
 void mul(BigTorus *res, int64_t a, const BigTorus *b, const ZModuleParams<BigTorus> *params);
 
+void mul_no_overlap(BigTorus *res, const BigInt *a, const BigTorus *b, const ZModuleParams<BigTorus> *params);
```

Comment:
add the doxygen comment to this function. Actually, I think that the comment of the next function is mis-placed.